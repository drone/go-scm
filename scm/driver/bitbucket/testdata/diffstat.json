{
  "pagelen": 500,
  "values": [
    {
      "status": "modified",
      "old": {
        "path": "CONTRIBUTING.md",
        "type": "commit_file",
        "links": {
          "self": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/atlaskit\/src\/dec26e0fe887167743c2b7e36531dedfeb6cd478\/CONTRIBUTING.md"
          }
        }
      },
      "lines_removed": 15,
      "lines_added": 15,
      "new": {
        "path": "CONTRIBUTING.md",
        "type": "commit_file",
        "links": {
          "self": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/atlaskit\/src\/425863f9dbe56d70c8dcdbf2e4e0805e85591fcc\/CONTRIBUTING.md"
          }
        }
      },
      "type": "diffstat"
    },
    {
      "status": "renamed",
      "old": {
        "path": "old-folder/CONTRIBUTING.md",
        "type": "commit_file",
        "links": {
          "self": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/atlaskit\/src\/dec26e0fe887167743c2b7e36531dedfeb6cd478\/CONTRIBUTING.md"
          }
        }
      },
      "lines_removed": 0,
      "lines_added": 0,
      "new": {
        "path": "new-folder/CONTRIBUTING.md",
        "type": "commit_file",
        "links": {
          "self": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/atlaskit\/src\/425863f9dbe56d70c8dcdbf2e4e0805e85591fcc\/CONTRIBUTING.md"
          }
        }
      },
      "type": "diffstat"
    },
    {
      "status": "removed",
      "old": {
        "path": "CONTRIBUTING.md",
        "type": "commit_file",
        "links": {
          "self": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/atlaskit\/src\/dec26e0fe887167743c2b7e36531dedfeb6cd478\/CONTRIBUTING.md"
          }
        }
      },
      "lines_removed": 15,
      "lines_added": 0,
      "new": {
        "path": "",
        "type": "commit_file",
        "links": {
          "self": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/atlaskit\/src\/425863f9dbe56d70c8dcdbf2e4e0805e85591fcc\/CONTRIBUTING.md"
          }
        }
      },
      "type": "diffstat"
    }
  ],
  "page": 1,
  "size": 1
}