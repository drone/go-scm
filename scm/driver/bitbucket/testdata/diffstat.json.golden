[
    {
        "Path": "CONTRIBUTING.md",
        "Added": false,
        "Renamed": false,
        "Deleted": false
    },
    {
        "Path": "new-folder/CONTRIBUTING.md",
        "Added": false,
        "Renamed": true,
        "Deleted": false,
        "PrevFilePath": "old-folder/CONTRIBUTING.md"
    },
    {
        "Path": "CONTRIBUTING.md",
        "Added": false,
        "Renamed": false,
        "Deleted": true
    }
]